samples/oauth_rails/VetsApiAppeals/app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,11 @@
+class PagesController < ApplicationController
+  def index
+    # No-Op
+  end
+
+  def login
+    session[:apikey] = params[:apikey] if params[:apikey].present?
```

Comment:
I thought about it but I think since the create requires an api key anyway, I didn't see much use in letting them clear it out - they can just supply a different API key if they want to do that for some reason.